### PR TITLE
NODE-5888 use user token with group name

### PR DESCRIPTION
**Pull request by dmironov-wallarm**
use user token + group_name instead of node token

### CODE REVIEW

File: test/e2e/run-chart-test.sh [hunk 1]
```diff
@@ -16,6 +16,9 @@
 
 KIND_LOG_LEVEL="1"
 
+# generate unique group name
+export NODE_GROUP_NAME="github-sidecar-$(tr -dc A-Za-z0-9 </dev/urandom | head -c 12; echo)"
```
Comment (alexorekhov) [inline test/e2e/run-chart-test.sh:20]: А точно sidecar?

Comment (alexorekhov) [inline test/e2e/run-chart-test.sh:20]: Are you sure there should be sidecar?

File: test/e2e/run-kind-e2e.sh [hunk 2]
```diff
@@ -60,6 +60,9 @@ SKIP_INGRESS_IMAGE_CREATION="${SKIP_INGRESS_IMAGE_CREATION:-false}"
 SKIP_E2E_IMAGE_CREATION="${SKIP_E2E_IMAGE_CREATION:=false}"
 SKIP_CLUSTER_CREATION="${SKIP_CLUSTER_CREATION:-false}"
 
+# generate unique group name
+export NODE_GROUP_NAME="github-sidecar-$(tr -dc A-Za-z0-9 </dev/urandom | head -c 12; echo)"
```
Comment (alexorekhov) [inline test/e2e/run-kind-e2e.sh:64]: And here

File: test/smoke/run.sh [hunk 3]
```diff
@@ -14,9 +14,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-#import functions
+# import functions
 source "${PWD}/test/smoke/functions.sh"
 
+# generate unique group name
+export NODE_GROUP_NAME="github-sidecar-$(tr -dc A-Za-z0-9 </dev/urandom | head -c 12; echo)"
```
Comment (alexorekhov) [inline test/smoke/run.sh:21]: Same here

File: test/e2e/run-e2e-suite.sh [hunk 4]
```diff
@@ -28,6 +28,9 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+# generate unique group name
+export NODE_GROUP_NAME="github-sidecar-$(tr -dc A-Za-z0-9 </dev/urandom | head -c 12; echo)"
```
[the remark this answers is not in the document]
Comment (dmironov-wallarm) [reply test/e2e/run-e2e-suite.sh]: done

File: test/e2e/run-kind-e2e.sh (hunk 2 above)
Comment (dmironov-wallarm) [reply test/e2e/run-kind-e2e.sh]: done

File: test/smoke/run.sh (hunk 3 above)
Comment (dmironov-wallarm) [reply test/smoke/run.sh]: done

File: test/e2e/run-chart-test.sh (hunk 1 above)
Comment (dmironov-wallarm) [reply test/e2e/run-chart-test.sh]: done

**Review by alexorekhov — APPROVED** [review]

**Pull request merged**